src/engines/pallet_engine/parser.rs

```diff
@@ -0,0 +1,913 @@
+#![allow(unused)]
+// This file is part of Substrate modified to fit pallet manipulation needs for pop-cli
```

Comment:
Do we use everything from here?

Reply:
Not yet. This is important when a future implementation will use in place editing of `RuntimeDeclaration` rather than inserting strings. It's basically the entire parser for `construct_runtime!`